src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```

Comment:
* Kindly use class-based components as it is a requirement for this milestone [Refer to the requirement here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.)